src/htmx.js

```diff
@@ -1232,6 +1234,18 @@ return (function () {
             return result;
         }
 
+        function consumeCSSSelector(tokens) {
+            var result = "";
+            if (tokens.length > 0 && tokens[0].match(COMBINED_SELECTOR_START)) {
+                tokens.shift();
+                result += consumeUntil(tokens, COMBINED_SELECTOR_END).trim();
+                tokens.shift();
+            } else {
+                result += consumeUntil(tokens, WHITESPACE_OR_COMMA);
```

Comment:
Same here. And please do not set `result` to empty at the beginning of the function.